xds/internal/xdsclient/bootstrap/bootstrap.go

```diff
@@ -86,6 +86,22 @@ type ServerConfig struct {
 	NodeProto proto.Message
 }
 
+// String returns the string representation of the ServerConfig.
+//
+// Note that it covers (almost) all the fields so the string can be used map
+// keys. Except NodeProto because NodeProto isn't used by federation (and this
+// is only used as key in federation).
```

Comment:
I don't understand the last line the comment:

```
// Except NodeProto because NodeProto isn't used by federation (and this
// is only used as key in federation)
```

You are saying it is not used by federation, but is used as a key in federation. Can you please clarify.

Reply:
Comment updated.